classes/install/Upgrade.inc.php

```diff
@@ -549,6 +549,23 @@ private function _transferSignoffData($signoffId, $queryId) {
 
 		$submissionFileDao->update('DELETE FROM signoffs WHERE signoff_id=?', array($signoffId));
 	}
+
+	/**
+	 * For 3.1.0 upgrade.  DefaultMenus Defaults
+	 */
+	function installDefaultNavigationMenus() {
+		$contextDao = Application::getContextDAO();
+		$navigationMenuDao = DAORegistry::getDAO('NavigationMenuDAO');
+
+		$contexts = $contextDao->getAll();
+		while ($context = $contexts->next()) {
+			$navigationMenuDao->installSettings($context->getId(), 'registry/navigationMenus.xml');
+		}
+
+		$navigationMenuDao->installSettings(CONTEXT_ID_NONE, 'registry/navigationMenus.xml');
+
+		return true;
+	}
```

Comment:
Could this go into `pkp-lib` since `registry/navigationMenus.xml` will be different for each app?

Reply:
I think it will be different - for example the Archive/Current MenuItems for OJS and Catalog MenuItem for OMP

Reply:
These items are defined in `registry/navigationMenus.xml`, right? The code above is the same for both apps...

Reply:
Yes you are right @NateWr. Done.